src/benchmarks/micro/libraries/System.Runtime.Extensions/Perf.Path.cs

```diff
@@ -15,6 +15,8 @@ public class Perf_Path
         private readonly string _testPath200 = PerfUtils.CreateString(200);
         private readonly string _testPath500 = PerfUtils.CreateString(500);
         private readonly string _testPath1000 = PerfUtils.CreateString(1000);
+        private readonly string _testPathNoRedundantSegments = "/home/user/runtime/src/coreclr/runtime/src/libraries/System.Private.CoreLib/src/System/IO/Path.cs";
+        private readonly string _testPathWithRedundantSegments = "/home/user/runtime/src/coreclr/runtime/src/libraries/System.Private.CoreLib/src/System/IO/..//./Path.cs";
```

Comment:
Did you consider adding rooted paths like `C:\ProgramData` and paths with flipped separators?

Reply:
I had a conversation with Adam a few minutes ago, we touched the topic of Windows, precisely.
I added a new commit, can you please take a look? It has comments describing the purpose of each new path.

Reply:
What about the flipped separators? It looks to me like a common case that is worth checking for regression, at least on windows.

Reply:
I really doubt flipped separators will cause a problem. We were calling `Path.IsDirectorySeparator` [before](https://github.com/dotnet/runtime/blob/1d9e50cb4735df46d3de0cee5791e97295eaf588/src/libraries/System.Private.CoreLib/src/System/IO/PathInternal.cs#L137), and we are still calling that.